pan_controller:
    type: dynamixel_hardware_interface/JointPositionController
    joint: pan_joint
    max_velocity: 3.2
    init_velocity: 1.2
    motor:
        id: 4
        init: 512
        min: 0
        max: 1023

tilt_controller:
    type: dynamixel_hardware_interface/JointPositionController
    joint: tilt_joint
    max_velocity: 3.2
    init_velocity: 1.2
    motor:
        id: 5
        init: 512
        min: 0
        max: 1023

multi_joint_c:
    type: dynamixel_hardware_interface/JointTrajectoryActionController
    joint_trajectory_action_node:
        min_velocity: 0.1
        constraints:
            goal_time: 0.25

